git-extra/PKGBUILD

```diff
@@ -1,11 +1,15 @@
 # Maintainer: nalla <nalla@hamal.uberspace.de>
 
-pkgname=('git-extra')
+_realname="git-extra"
+pkgbase="mingw-w64-${_realname}"
+pkgname=($_realname
+        "${MINGW_PACKAGE_PREFIX}-${_realname}")
 _ver_base=1.1
 pkgver=1.1.614.d551cf0cc
 pkgrel=1
 pkgdesc="Git for Windows extra files"
-arch=('i686' 'x86_64')
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'clangarm64')
```

Comment:
This is consistent with how the MSYS2 team works. For example: https://github.com/msys2/MINGW-packages/blob/c33af64b6b5ac989b2611da1aca6b5a6fb77c618/mingw-w64-curl/PKGBUILD#L11-L12